src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/TaskListTabPanel.tsx

```diff
@@ -144,7 +144,11 @@ const TaskListTabPanel = (props: TaskListTabPanelProps) => {
     [assignees]
   );
 
-  if (isLoading || assigneeIsLoading || !auth.user || !team) return <LoadingIndicator />;
+  const failMessageStyle: SxProps = { py: '25px', textAlign: 'center' };
+
+  if (isLoading || assigneeIsLoading || !auth.user) return <LoadingIndicator />;
+  if (!team)
+    return <>{value === index && <Typography sx={failMessageStyle}>No team assigned to this project!</Typography>}</>;
```

Comment:
I think this syntax is confusing, lets just make a component called like NoTeamErrorMessage and then return that if theres no team maybe